cmd/metacache-bucket.go

```diff
@@ -87,7 +87,8 @@ func loadBucketMetaCache(ctx context.Context, bucket string) (*bucketMetacache,
 		select {
 		case <-ctx.Done():
 			return nil, ctx.Err()
-		case <-time.After(250 * time.Millisecond):
+		default:
+			time.Sleep(250 * time.Millisecond)
```

Comment:
This has the sideeffect that canceling will delay up to 250ms. No huge deal AFAICT.

Reply:
Yeah that's why I thought its simpler this way.